content/guides/getting-started.md

```diff
@@ -109,7 +109,7 @@ __src/index.js__
 +
   function component() {
     var element = document.createElement('div');
-  
+
 -   // Lodash, currently included via a script, is required for this line to work
```

Comment:
This comment was on purpose, as it shows the user that in this case lodash is imported through a `<script>` tag, whereas later on it's imported by webpack.

Would you mind leaving it in (or maybe clarify it)?